src/main/java/org/carlmontrobotics/Constants.java

```diff
@@ -25,12 +24,12 @@ public static final class Arm {
 		public static final double ampAngle = .3;
 		public static final double speakerAngle = .4;
 		//if needed
-//		public static final trapAngle = 80;
+		//public static final trapAngle = 80;
 		
 		//Motor Controllers: pid, FF
 		public static final double[] pidVals = new double[] { 0.1, 0.0, 0.1 };
 		// Feedforward
-		public static final double kS = 0.1;
+		public static final double kS = 0.1; 
 		public static final double kG = 0.1;
 		public static final double kV = 0.1;
 		public static final double kA = 0.1;
```

Comment:
For consistency if you made pid values into an array do the same to the feedforward values. Also aren't you supposed to work on shooter not arm?